contrib/gorilla/mux/mux_test.go

```diff
@@ -11,70 +13,76 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestHttpTracer200(t *testing.T) {
-	assert := assert.New(t)
-	mt := mocktracer.Start()
-	defer mt.Stop()
+var httpTests = []struct {
+	responseCode int
```

Comment:
I find it would be more succinct to call these `code`, `method` and `url`. They are all HTTP-related, so the "http" prefix is not needed. As for code, it's the same in the standard library, it's called "code" instead of "responseCode".